packages/manager/modules/office/src/office.routing.js

```diff
@@ -16,6 +16,44 @@ export default /* @ngInject */ ($stateProvider) => {
     resolve: {
       ...ListLayoutHelper.stateResolves,
       apiPath: () => '/license/office',
+      postpaidLicenses: /* @ngInject */ (iceberg) =>
+        iceberg('/license/office')
+          .query()
+          .expand('CachedObjectList-Pages')
+          .limit(5000)
+          .execute(null, true)
+          .$promise.then(({ data }) => data),
+      prepaidLicenses: /* @ngInject */ (iceberg) =>
+        iceberg('/license/officePrepaid')
+          .query()
+          .expand('CachedObjectList-Pages')
+          .limit(5000)
+          .execute(null, true)
+          .$promise.then(({ data }) => data),
+      resources: /* @ngInject */ ($http, postpaidLicenses, prepaidLicenses) =>
+        $http
+          .get('/service', {
+            params: {
+              external: false,
+              type: '/license/office',
+            },
+            serviceType: 'aapi',
+          })
+          .then(({ data }) => {
+            const licenseDetails = data.map((license) => {
+              const licenseDetail = postpaidLicenses?.some(
+                (option) => option.serviceName === license.serviceName,
+              )
+                ? postpaidLicenses.find(
+                    (option) => option.serviceName === license.serviceName,
+                  )
+                : prepaidLicenses.find(
+                    (option) => option.serviceName === license.serviceName,
+                  );
+              return { ...licenseDetail };
+            });
+            return { data: licenseDetails };
+          }),
```

Comment:
**question:** Why fetching the same data twice? Once using `iceberg` and again from `2API`. Can we avoid over-fetching and use only one call?

Reply:
done Anoop, pulled other details by modifying the response of api call in 2api